app/Http/Controllers/RegisterController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Input;
+use Validator;
+use Redirect;
+use App\Models\Register;
+use Auth;
+
+class RegisterController extends Controller
+{
+    public function index(){
+        // echo 'Welcome';
+        $data=Input::except(array('_token'));
+
+        // var_dump($data);
+
+        $rule = array(
+            'username'  =>  'required',
+            'email'     =>  'required|email',
+            'password'  =>  'required|min:6',
+            'cpassword' =>  'required|same:password'
+        );
+
+        $message = array(
+            'cpassword.required' => 'the confirm password is required',
+            'cpassword.min'     => 'the confirm password must be at least 6 characters',
+            'cpassword.same'     => 'the confirm password and password must same'
+        );
+
+        $validator = Validator::make($data, $rule, $message);
+
+        if($validator->fails()){
+            return Redirect::to('register')->withErrors($validator);
+        } else{
```

Comment:
- [ ] Pls use `Form Request` instead of direct validation
- [ ] Use [rule-confirmed](https://laravel.com/docs/8.x/validation#rule-confirmed) instead.